app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -24,19 +27,42 @@ public ProductService(
         this.productRepository = productRepository;
     }
 
+    /**
+     * 모든 상품 목록을 반환합니다.
+     */
     public List<Product> getProducts() {
         return productRepository.findAll();
     }
 
+    /**
+     * 아이디로 상품을 찾아서 상품을 반환합니다.
+     *
+     * @param id 는 상품의 아이디 입니다.
+     * @throws ProductNotFoundException 는 상품 아이디가 존재하지 않을 때 던집니다.
+     */
+
     public Product getProduct(Long id) {
         return findProduct(id);
     }
 
+    /**
+     * 새로운 상품 데이터를 상품 목록에 저장합니다.
+     * @param productData 는 등록하려는 상품 데이터 입니다.
```

Comment:
"상품 목록"을 사용하고 있다는 기밀이 여기에서 유출되면 좋지 않습니다.
```suggestion
     * 새로운 상품 데이터를 저장하고, 저장된 상품 정보를 리턴합니다.
     * @param productData 저장하려는 상품 데이터
```

Reply:
생각지도 못했습니다..! 👍🏻 감사합니다 💯 🥇